iphone/Classes/TiUITableView.m

```diff
@@ -2116,7 +2140,11 @@ - (UITableViewCell *)tableView:(UITableView *)ourTableView cellForRowAtIndexPath
     if ([TiUtils isIOS8OrGreater] && (tableview == ourTableView)) {
         [cell setLayoutMargins:UIEdgeInsetsZero];
     }
-	
+    
+    if (rowSeparatorInsets.left && rowSeparatorInsets.right != 0) {
```

Comment:
Yeah i went or not and , my bad. I do the check because if the variable is not set that way whatever  
tableSeparatorInsets is set too the cell insets will be the same. That way we keep the original behavior as well as the new one.